packages-next/auth/src/components/SigninContainer.tsx

```diff
@@ -22,7 +22,7 @@ export const SigninContainer = ({ children }: SigninContainerProps) => {
       <Box
         css={{
           background: colors.background,
-          minWidth: 600,
+          width: 600,
```

Comment:
changed this to have uniform width between screens.
this component is only used by 3 components, Welcome, InitPage and Signin 

Before the change InitPage and Signin were the same length coincidentally due to having similar contents. However Welcome was far too large due to much copy. 

This change resolves this issue.